build_tools/docker/Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN apt-get update && \
 
 ######## Bazel ########
 WORKDIR /install-bazel
-ARG BAZEL_VERSION=5.2.0
+ARG BAZEL_VERSION=6.0.0
```

Comment:
My macros function `gen_python_package` wasn't work on lower bazel version, So I update it to 6.0.0.